datasets/census_bureau_acs/cbsa_2019_1yr/pipeline.yaml

```diff
@@ -0,0 +1,825 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+
+  - type: bigquery_table
+    table_id: cbsa_2019_1yr
+    description: "CBSA 2019 1 year report table"
+
+dag:
+  airflow_version: 1
+  initialize:
+    dag_id: cbsa_2019_1yr
+    default_args:
+      owner: "Google"
+      depends_on_past: False
+      start_date: "2021-03-01"
+    max_active_runs: 1
+    schedule_interval: "@once"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "KubernetesPodOperator"
+      description: "Run CSV transform within kubernetes pod"
+      args:
+        task_id: "cbsa_2019_1yr_transform_csv"
+        startup_timeout_seconds: 600
+        name: "cbsa_2019_1yr"
+        namespace: "default"
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+                - matchExpressions:
+                    - key: cloud.google.com/gke-nodepool
+                      operator: In
+                      values:
+                        - "pool-e2-standard-4"
+        image_pull_policy: "Always"
+        image: "{{ var.json.census_bureau_acs.container_registry.run_csv_transform_kub }}"
+        env_vars:
+          SOURCE_URL: "https://api.census.gov/data/2019/acs/acs+year_report+?get=NAME,+key[0:-3]+_+key[-3:]+E&for=+api_naming_convention+:*&key=550e53635053be51754b09b5e9f5009c94aa0586"
```

Comment:
Let's replace the `key` ranges with more descriptive, parameterized names

Reply:
done